docker/base/Dockerfile

```diff
@@ -47,6 +47,7 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
 RUN ln -s /usr/lib/x86_64-linux-musl/libc.so /lib/libc.musl-x86_64.so.1
 
 # install git-lfs
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6B05F25D762E3157
```

Comment:
can you combine this with the line below into a single `RUN`?

Reply:
done. 

@rokroskar: note that git-lfs is now in the 2.7.x range, should we update that, too?

Reply:
@rokroskar : so, let's switch to `2.7.1`, no?